### PR TITLE
Swipe list view

### PR DESCRIPTION
**Pull request by soriasayer**
In this PR swipe list, delete and edit features are implemented.

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -181,11 +181,12 @@ export default class App extends Component {
   }
 
   render() {
+
     return (
       <Provider store={store} >
         <PersistGate persistor={persistedStore} >
             <View style={{flex: 1}} accessible>
-              <AppStatusBar backgroundColor={orange} barStyle="light-content" />
+              <AppStatusBar backgroundColor={'#5bba99'} barStyle="light-content" />
```
Comment (aemal) [inline App.js]: @soriasayer move this color code in a variable from a central file

File: components/AddDeck.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { white, gray, lightGray } from '../utils/colors'
 import {widthPercentageToDP as wp, heightPercentageToDP as hp} from 'react-native-responsive-screen'
 import { connect } from 'react-redux'
 import { addDeckTitle } from '../actions/deck'
+import { generateUID } from '../utils/helpers'
```
Comment (aemal) [inline components/AddDeck.js]: @soriasayer UUID not UID

File: components/AddQuestion.js [hunk 3]
```diff
@@ -16,11 +15,10 @@ class AddQuestion extends Component {
     handlePress = () => {
         const { question, answer } = this.state
         const { dispatch, navigation } = this.props
-        const { deckTitle } = this.props.route.params
+        const { deckId } = this.props.route.params
```
Comment (aemal) [inline components/AddQuestion.js]: @soriasayer destructure this.props.route in the line above this

File: components/Decks.js [hunk 4]
```diff
@@ -1,34 +1,132 @@
 import React, { Component } from 'react'
-import { FlatList, ScrollView, SafeAreaView, View } from 'react-native'
+import { FlatList, View, Text, StyleSheet, TouchableHighlight, Alert } from 'react-native'
 import FlashCards from './FlashCards'
 import { getData } from './FlashCards'
 import { connect } from 'react-redux'
+import { SwipeListView, SwipeRow } from 'react-native-swipe-list-view'
+import {widthPercentageToDP as wp, heightPercentageToDP as hp} from 'react-native-responsive-screen'
```
Comment (aemal) [inline components/Decks.js]: @soriasayer fix your prettier

File: components/Decks.js [hunk 5]
```diff
@@ -1,34 +1,132 @@
...
+    icons: {
+        alignSelf: 'center',
+        color: white,
+    }
+} )
+
 function mapStateToProps({decks}) {
-    const data = Object.keys(decks).map(deck => decks[deck])
-    
+    const data = Object.keys( decks ).map( deck => decks[ deck ] )
```
Comment (aemal) [inline components/Decks.js]: @soriasayer fix your prettier

File: components/FlashCards.js [hunk 6]
```diff
@@ -1,85 +1,172 @@
...
+        <TextInput
+        autoFocus={true}
+        maxLength={200}
+        defaultValue={textInput}
+        onChangeText={(text) => this.setState({newTitle: text})} 
+        style={styles.input}/>
+        <TouchableHighlight 
+         style={styles.button} 
+         onPress={this.handleOnPress} 
+         underlayColor={'#f1f1f1'}>
```
Comment (aemal) [inline components/FlashCards.js]: @soriasayer move the color in a variable

File: components/Question.js [hunk 7]
```diff
@@ -140,20 +140,19 @@ const styles = StyleSheet.create({
...
-    
+} )
+
+function mapStateToProps( {decks, counter}, {route} ) {
+    const {deck} = route.params
+
     return {
-        questions: decks[deck].questions,
-        currentQuestion: decks[deck].questions[counter],
+        questions: decks[ deck ].questions,
```
Comment (aemal) [inline components/Question.js]: @soriasayer prettier

**Comment by soriasayer** [comment]
All comments fixed.

**Pull request merged**